### ISSUE TITLE
VB: Dictionary access operator formats incorrectly

### ISSUE THREAD

**Issue by DustinCampbell** [dotnet/roslyn#8258]
```VB
Class C
    Public Shared Sub AutoFormatSample()

        ' Automatic Code Formatting in VB.NET
        ' Problem with ! dictionary access operator

        Dim dict As New Dictionary(Of String, String)
        dict.Add("Apple", "Green")
        dict.Add("Orange", "Orange")
        dict.Add("Banana", "Yellow")

        Dim x As String = (New Dictionary(Of String, String)(dict)) !Banana 'added space in front of "!"

        With dict

            !Apple = "Red"
            Dim multiColors = !Apple &!Orange &!Banana 'missing space between "&" and "!"
            If!Banana = "Yellow" Then 'missing space between "If" and "!"
                !Banana = "Green"
            End If

        End With

    End Sub
End Class
```

**Comment by DustinCampbell** [comment]
This was reported through VS Feedback
